scripts/datetimes.py

```diff
@@ -39,6 +92,46 @@ def format_rfc_3339_nz_midnight_datetime_string(datetime_object: datetime) -> st
     return format_rfc_3339_datetime_string(datetime_utc)
 
 
+def convert_utc_to_nz(datetime_object: datetime) -> datetime:
```

Comment:
NIT function name clarity 

```suggestion
def convert_utc_to_nz_time(datetime_object: datetime) -> datetime:
```